Mathlib/Order/LiminfLimsup.lean

```diff
@@ -294,6 +294,66 @@ theorem IsCobounded.mono (h : f ≤ g) : f.IsCobounded r → g.IsCobounded r
 
 end Relation
 
+section add_and_sum
+
+open Filter BigOperators Set
+
+variable {α : Type*} {f : Filter α} [NeBot f]
+variable {R : Type*} [Preorder R]
+
+lemma isBoundedUnder_ge_add [Add R]
+    [CovariantClass R R (fun a b ↦ a + b) (· ≤ ·)] [CovariantClass R R (fun a b ↦ b + a) (· ≤ ·)]
```

Comment:
Why do we need the `ge` version?

Reply:
To prove boundedness from below of sums assuming the boundedness from below of the terms of the sums. (The `le` versions achieve the same for boundedness from _above_).